live-examples/js-examples/map/map.js

```diff
@@ -0,0 +1,17 @@
+const MAP = new Map();
```

Comment:
We don't usually uppercase variables. Please see https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#variable-naming for some guidelines on variable naming.

Reply:
Ah, I read that. I'm used to uppercase variables for constant declarations. I changed it to match the variable naming convention for this repository. 836c2c5 should address this.